### DISCUSSION TITLE
Next.js 13 : New <Image/> and TailwindCss

### DISCUSSION THREAD

**Discussion by jamiviz** [vercel/next.js#41888] [Help]
Hello , i just upgraded my project to Nextjs v13 and i have a problem with Image and Tailwind .
Before V13 i putted the Image in a div with relative postion and fixe the width and height in the parent like : 

```
 <div className="relative h-[250px] w-full ">
        <Image src={img}   alt="1" layout="fill" objectFit="cover" />
 </div>

```
But now i get an error : Error: Image with src... is missing required "width" property.

Any solution please ?

**Comment by SimonVillage** [comment]
`layout` and `objectFit` don't exist as props. Add `fill` prop to `Image` and `object-cover` class to parent.

https://nextjs.org/docs/api-reference/next/image#fill

**Reply by blizkyone** [reply]
I did and the image still behaves as if object contain is being used

` <div className="relative h-screen w-full overflow-hidden object-cover">
        <Image
          className="z-0"
          src="/laundry.jpg"
          fill
          priority={true}
        /></div>`

**Reply by blizkyone** [reply]
Solved by adding object-over to the Image component className, not the parent div

**Reply by jamiviz** [reply]
@blizkyone That work for me too , thanks !

**Comment by jamiviz** [comment]
Thanks for replay but it still dont work and ask me for width and height :

```
<div className="relative h-[250px] w-full  fill="true"  style={{objectFit: "cover"}}	  ">
        <Image src={img}   alt="1"/>
 </div>
```

**Reply by PaulHaze** [reply]
I think the 'fill' prop needs to go onto the Image component, not its parent div. Also, Tailwind has an 'object-cover' utility class. So the final code should look like: 
```
<div className="relative h-[250px] w-full object-cover">
        <Image src={img}   alt="1" fill/>
 </div>
```

**Reply by jamiviz** [reply]
the following code work , i have no more errors ,but the object-cover object-contain in the parent have no effect !
```
<div className="relative h-[250px] w-full object-contain">
        <Image src={img}   alt="1" fill/>
 </div>

```

**Reply by blizkyone** [reply]
Add it to the Image component className string, not the parent div

**Reply by dchien234** [reply]
I think their docs need update to highlight the part that they prefer `style` & `className` over the existing `layout` prop now.

https://nextjs.org/docs/api-reference/next/image

**Reply by SalahAdDin** [reply]
I can't make it work yet.

**Comment by xcollantes** [comment]
CSS: 

```css 
.imageContainer {
  width: 100%;
}

.image {
  max-width: 800px;
  margin: auto;
  object-fit: contain;
  width: 100% !important;
  position: relative !important;
}
```

TSX/JSX: 

```js
import mycss from "../file/where/css"
<div className={mycss}>
            <img
              className={utilsStyles.image}
          </div>
```

**Reply by SalahAdDin** [reply]
But, this is not `tailwindcss` man.

**Reply by RicardoValdovinos** [reply]
Not only is it not `tailwindcss` but it's not even using Next.js `<Image />` on a thread discussing issues using the two together. What was the point of this suggestion?
